### ISSUE TITLE
Crypto.Random: Module has no attribute "get_random_bytes"

### ISSUE THREAD

**Issue by muendelezaji** [python/mypy#7934]
* Are you reporting a bug, or opening a feature request?
Bug

Prerequisite: `pip install pycrypto`

Example code:
```py
# test_27.py
# Python 2.7 annotation style
from Crypto import Random
b27 = Random.get_random_bytes(10) # type: bytes
```
```py
# test_35.py
# Python 3.5+ annotation style
from Crypto import Random
b27 = Random.get_random_bytes(10) # type: bytes
b35: bytes = Random.get_random_bytes(10)
```

* What is the actual behavior/output?
```
$ mypy --py2 test_27.py
test_27.py:4: error: Module has no attribute "get_random_bytes"

$ mypy test_35.py
test_35.py:4: error: Module has no attribute "get_random_bytes"
test_35.py:5: error: Module has no attribute "get_random_bytes"
```

* What is the behavior/output you expect?
Expected no errors as per [package source](https://github.com/dlitz/pycrypto/blob/v2.6.1/lib/Crypto/Random/__init__.py#L39)

* What are the versions of mypy and Python you are using?
`mypy==0.740` and `Python==3.7.4`

* What are the mypy flags you are using?
No flags nor custom config

Is this expected behaviour (maybe missing a flag)?

**Comment by srittau** [comment]
This is a typeshed issue (could someone move this issue?) PR in typeshed welcome!
